sentry_sdk/tracing_utils.py

```diff
@@ -506,7 +508,28 @@ def span_id(self, value):
     @property
     def dynamic_sampling_context(self):
         # type: () -> Optional[Dict[str, Any]]
-        return self.baggage.dynamic_sampling_context() if self.baggage else None
+        return self.get_baggage().dynamic_sampling_context()
+
+    def get_traceparent(self):
```

Comment:
Nit: making this consistent with the `Span` methods
```suggestion
    def to_traceparent(self):
```

Reply:
done